### ISSUE TITLE
`alphabetical-stylesheets` throws error if using variable for stylesheet

### ISSUE THREAD

**Issue by EvNaverniouk** [Eight-Solutions/eslint-plugin-cumul8#4]
The following code:

```
import {typography} from 'react-native-material-design-styles';

const textStyle = StyleSheet.create(typography);
```

Will throw the following ESLint error:

```
TypeError: Cannot read property 'length' of undefined
    at EventEmitter.CallExpression (\eslint-plugin-cumul8\lib\rules\alphabetical-stylesheets.js:64:25)
    at emitOne (events.js:95:20)
    at EventEmitter.emit (events.js:182:7)
```

@jordanschalm Could you look into this?

**Comment by jordanschalm** [comment]
#5

**Comment by EvNaverniouk** [comment]
Fixed in v1.4.2
